category_theory/examples/measurable_space.lean

```diff
@@ -16,7 +16,7 @@ namespace category_theory.examples
 
 @[reducible] def Meas : Type (u+1) := bundled measurable_space
 
-instance (x : Meas) : measurable_space x := x.str
+instance (x : Meas) : measurable_space x := x.inst
```

Comment:
Would it be possible to autogenerate these instances? Everytime we bundle a class we want an instance like this. This probably means that `bundled` has to become `meta`. I don't know. But I think it would take away one of those "minor annoyances" if all these instances would just be there, automagically.